### ISSUE TITLE
Reworking the WoTR events

### ISSUE THREAD

**Issue by Lonhaldar** [Middle-Earth-Project-Team/mod#145]
- Replacing the current events with new ones
- Reworking all the chain from the start (adding more depths, resolving bugs, ...)

**Comment by Rigmored** [comment]
Feel free to steal from here
MEM Dropbox\Team Members Cubbys\ReditRigmored\new wotr event

**Comment by Lonhaldar** [comment]
Yeah, i planned to use some of your work :) I'll try to create something clean and easy to change/find. If you've got some suggestions about missing stuff from the old versions, just post here what you want to see. I'll try to explain what i'm doing step by step too on my side.

**Comment by Rigmored** [comment]
Perfect, will think about it :)

**Comment by Lonhaldar** [comment]
Yeah, sounds interesting.

What I'll do is several files, each ones linked with some important areas/period : 
- The Shire to Rivendell trip + The Council of Elrond : mep_wotr_shiretorivendell.txt
- The Rivendell to Amon Hen trip : mep_wotr_rivendelltorohan.txt
- Rohan events : mep_wotr_rohan_isengard_wars.txt
- Gondor events : mep_wotr_ gondor.txt
- North events (Wars in Rhovanion for Dale/Erebor) : mep_wotr_northernwars.txt
- The Quest of Frodo (Ithilien + Mordor) = mep_wotr_frodo.txt
- The events of the Shire (Saruman men + battle of Bywater) + departure of ringbearers : mep_wotr_ending.txt

It'll then be more easy to change some aspects in the future!

**Comment by Lonhaldar** [comment]
Current Status : 
- First events in Rohan/Gondor done
- Travel of the Fellowship until Rivendell done
